spec/unit/models/project_spec.rb

```diff
@@ -49,6 +61,80 @@
           .to raise_error(/Cannot transfer output stage from foo to bar/)
       end
     end
+
+    it 'creates output stage with the same prefix as in the original project' do
+      expect(GoodData::AdsOutputStage).to receive(:create)
+        .with(hash_including(output_stage_prefix: original_prefix))
+      subject.transfer_output_stage(from_project, to_project, {})
+    end
+
+    context 'when ads_output_stage_prefix is specified' do
```

Comment:
warning: Similar code found in project_spec.rb:71, project_spec.rb:83 (mass = 60) (flay)